openshift/prometheus-flatliner-deployment-template.yaml

```diff
@@ -161,3 +161,27 @@ objects:
           name: ${APPLICATION_NAME}:latest
       type: ImageChange
     - type: ConfigChange
+
```

Comment:
For future PR's - try to only include changes that are relevant to the PR.

* adding the service and route
* changing the timestamp name

these 2 are a PR on it's own. It would up your number of PRs as well :)

Reply:
Got it. Thanks